boards/renesas/da14695_dk_usb/da14695_dk_usb.yaml

```diff
@@ -9,9 +9,18 @@ toolchain:
   - xtools
 supported:
   - arduino_gpio
+  - counter
   - gpio
   - hwinfo
   - watchdog
   - i2c
   - spi
   - usb_device
+  - rtc
+  - crypto
+  - dma
+  - mipi_dbi
+  - display
+  - memc
```

Comment:
It does not belong in this PR and you need to remove them from this commit. (These are test features)